Sources/MapboxSearch/PublicAPI/MapboxSearchVersion.swift

```diff
@@ -1,2 +1,2 @@
 /// Mapbox Search SDK version variable
-public let mapboxSearchSDKVersion = "1.0.0-beta.27"
```

Comment:
@sarochych I’m not sure about this one though as I didn’t check the release process. Feel free to just commit directly to change this.